critiquebrainz/db/__init__.py

```diff
@@ -2,7 +2,7 @@
 from sqlalchemy.pool import NullPool
 
 # This value must be incremented after schema changes on exported tables!
-SCHEMA_VERSION = 1
+SCHEMA_VERSION = 6
```

Comment:
I see. I didn't quite understand where that number came from at first, which is why I asked. It should be the same as it was before actually (`6`). It needs to be like that so that there are no compatibility issues when someone tries to import data dump generated from an old schema.